ha/virsh/tests/resource_iscsi_target_test.sh

```diff
@@ -0,0 +1,96 @@
+# shellcheck shell=bash
+
+# shellcheck disable=SC1090
+. "$(dirname "$0")/helper/test_helper.sh"
+# shellcheck disable=SC1090
+. "$(dirname "$0")/../vm_utils.sh"
+
+install_targetcli() {
+  run_in_all_nodes "DEBIAN_FRONTEND=noninteractive sudo apt-get install -y targetcli-fb >/dev/null"
+}
+
+oneTimeSetUp() {
+  get_network_data_nic1
+  install_targetcli
+
+  readonly TARGET_RESOURCE_NAME="iscsi-target-rsc"
+  readonly LUN_RESOURCE_NAME="iscsi-lun0-rsc"
+  readonly IQN="iqn.2022-02.com.test"
+  readonly DEVICE="/dev/mapper/mpatha"
+  readonly LUN="0"
+}
+
+configure_cluster_properties() {
+  run_command_in_node "${IP_VM01}" "sudo crm configure property stonith-enabled=off"
+}
+
+configure_resources() {
+  run_command_in_node "${IP_VM01}" "sudo crm configure<<EOF
+primitive ${TARGET_RESOURCE_NAME} iSCSITarget \
+	  params implementation=lio-t iqn=\"${IQN}:target\"
+primitive ${LUN_RESOURCE_NAME} iSCSILogicalUnit \
+	  params implementation=lio-t target_iqn=\"${IQN}:target\" path=\"${DEVICE}\" lun=${LUN}
+group iscsi-target-group ${TARGET_RESOURCE_NAME} ${LUN_RESOURCE_NAME}
+commit
+EOF"
+}
+
+test_iscsi_target_resources_are_started() {
+  configure_cluster_properties
+  configure_resources
+
+  # Wait for changes to be applied and check if the resource are started
+  sleep 5
+  cluster_status=$(run_command_in_node "${IP_VM01}" "sudo crm status")
+  echo "${cluster_status}" | grep "${TARGET_RESOURCE_NAME}" | grep Started
+  assertTrue $?
+  echo "${cluster_status}" | grep "${LUN_RESOURCE_NAME}" | grep Started
+  assertTrue $?
```

Comment:
I recently proposed a set of PRs to move away from performing these assertions without information for the OCI tests.

The reasoning behind it is that, shall the test fail, one cannot easily infer why it did so nor even speculate what is going on without at least opening the test scripts and/or trying to reproduce the failure.

Would you think adding more info here could help in this matter as well?

For instance, you could do something like `assertTrue "Could not infer ${LUN_RESOURCE_NAME} readiness from status output:\n${cluster_status}" $?`

Reply:
That's indeed something to we should improve. I'll apply those changes in this PR and update the other tests when I touch them again.